test/ir_Airwell_test.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 David Conran
+
+#include "IRac.h"
+#include "IRrecv.h"
+#include "IRrecv_test.h"
+#include "IRsend.h"
+#include "IRsend_test.h"
+#include "gtest/gtest.h"
+
+// Tests for decodeAirwell().
+
+// Ref: https://github.com/crankyoldgit/IRremoteESP8266/issues/1069
+// Data from:
+//   https://github.com/crankyoldgit/IRremoteESP8266/issues/1069#issuecomment-604293514
+TEST(TestDecodeAirwell, RealExample) {
+  IRsendTest irsend(kGpioUnused);
+  IRrecv irrecv(kGpioUnused);
+  // ON / OFF / 25 degres heat
```

Comment:
spelling: degrees

Reply:
Yep. That will teach me to cut & paste from a user's comment.

Fixed.